packages/block-editor/src/components/inserter/menu.js

```diff
@@ -341,4 +341,16 @@ function InserterMenu(
 	);
 }
 
-export default forwardRef( InserterMenu );
+export const PrivateInserterMenu = forwardRef( InserterMenu );
```

Comment:
As far as I've seen the inserter menu component is:

- used exclusively in the block editor package
- used exclusively in the inserter component in the block editor package
- not specifically exported anywhere

What do we need the private component for?

Reply:
I thought about it too, and I think it would be fine to not create a private one. Sometimes though we can decide to make some existing components public ([recent example here](https://github.com/WordPress/gutenberg/pull/61529/)). With that in mind, it would be clear that some props are meant to remain private..